department-of-veterans-affairs/va.gov-cms
# VACMS-3923: Limit qa's in FAQ panel to 10.

## Description
See #3923 

## Testing done
Visual / behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/104352428-ab611b80-54d4-11eb-8044-c10c4d2fadde.png)

## QA steps
- As an administrator, go to `/node/15051/edit` and open the FAQs panel.
- Visually verify that the `Add Q&A` button does not appear.
- Remove one Q&A paragraph, and visually verify that the `Add Q&A` button does appear.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
